maisim/maisim.Desktop/maisimGameDesktop.cs

```diff
@@ -5,6 +5,17 @@ namespace maisim.Desktop
 {
     public class maisimGameDesktop : maisimGame
     {
+        public override void SetHost(GameHost host)
+        {
+            base.SetHost(host);
+            var desktopWindow = (SDL2DesktopWindow)host.Window;
+
+            desktopWindow.Title = "maisim";
+#if DEBUG
+            desktopWindow.Title += " development";
```

Comment:
Please use `DebugUtils.IsDebugBuild` instead of relying on preprocessor vars for this.